### PR TITLE
updated GitBook to Docusaurus

### PR DESCRIPTION
**Pull request by ghost**
Updates: replace GitBook OSS docs with Docusaurus platform

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,65 +4,34 @@
...
 
-# Getting Started
+**Do you have 5 minutes?** That is how long it takes to scaffold your first app. Try it now!
 
-- See the [Full README.md in docs](docs/README.md) or visit our [gitbook documents].
+## electrode quick start
 
-- [Getting Started with Electrode]
+```bash
+npm install -g xclap-cli
```
Comment (jchip) [inline README.md:24]: these instructions don't work.

```
mkdir my-app
cd my-app
npm init @xarc/app
npm install
npm run dev
```

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    [
+      '@docusaurus/preset-classic',
+      {
+        docs: {
+          // It is recommended to set document id as docs home page (`docs/` path).
+          homePageId: 'introduction',
+          sidebarPath: require.resolve('./sidebars.js'),
+          // Please change this to your repo.
+          editUrl:
+            'https://gecgithub01.walmart.com/electrode/docs',
```
Comment (datvong-wm) [inline docs/docusaurus.config.js:76]: internal

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        },
+        blog: {
+          showReadingTime: true,
+          // Please change this to your repo.
+          editUrl:
+            'https://gecgithub01.walmart.com/electrode/docs',
```
Comment (datvong-wm) [inline docs/docusaurus.config.js:82]: update url

File: README.md [hunk 4]
```diff
@@ -4,65 +4,36 @@
...
-> Built at @WalmartLabs powering the <http://www.walmart.com> eCommerce site, Electrode is a platform for building small to large scale universal ReactJS/NodeJS web applications with a standardized structure, best practices, and modern technologies baked in. Electrode focuses on performance, component reusability, and simple deployment to multiple cloud providers—so you can focus on what makes your app unique.
+Electrode Web is a rapid application development framework; a full stack, end-to-end platform for developing, deploying, and maintaining JavaScript applications at Walmart.
 
-**It takes 5 minutes and a single command to get your own Universal React app to customize:**
+:::warning
+This framework is for the open-source version of electrode.
 
-![Hello from Electrode][hello-electrode]
+If you are a Walmart developer, please see the internal electrode X documentation site.
+:::
```
Comment (jchip) [inline README.md:17]: We maintain additional code to support internal systems.  If you are a Walmart developer, please check internal docs to get started.

File: README.md [hunk 5]
```diff
@@ -4,65 +4,36 @@
...
-
-- See [What is Electrode] for more details on features and technologies supported.
-
-# Documentation
-
-The documents for Electrode are maintained as [gitbook documents] in the [docs](/docs) directory.
-
-Check [gitbook] for more info about gitbook.
-
-# Contributing
```
Comment (jchip) [inline README.md:34]: we should keep contributing section

File: README.md [hunk 6]
```diff
@@ -4,65 +4,36 @@
...
 
-### Building large scale universal React Web Applications
+Electrode was developed by Walmart Labs in 2016 as an internal rapid application development framework for Walmart. Electrode is one of the technology engines powering the <http://www.walmart.com> eCommerce website.
 
-> Built at @WalmartLabs powering the <http://www.walmart.com> eCommerce site, Electrode is a platform for building small to large scale universal ReactJS/NodeJS web applications with a standardized structure, best practices, and modern technologies baked in. Electrode focuses on performance, component reusability, and simple deployment to multiple cloud providers—so you can focus on what makes your app unique.
+Electrode Web is a rapid application development framework; a full stack, end-to-end platform for developing, deploying, and maintaining JavaScript applications at Walmart.
 
-**It takes 5 minutes and a single command to get your own Universal React app to customize:**
+:::warning
+This framework is for the open-source version of electrode.
```
Comment (jchip) [inline README.md:14]: this makes it sound like we have two versions.  we don't.

File: README.md [hunk 7]
```diff
@@ -4,65 +4,40 @@
 <b>Electrode</b>
 </p>
 
-[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](https://gitter.im/electrode-io/electrode)
-[![Build Status][travis-image]][travis-url] [![devDependencies Status][daviddm-image]][daviddm-url]
+# Building large scale universal React Web Applications
 
-### Building large scale universal React Web Applications
+Electrode Web is a rapid application development framework; a full stack, end-to-end platform for developing, deploying, and maintaining JavaScript applications at Walmart, afor the <http://www.walmart.com> eCommerce website.
```
Comment (jchip) [inline README.md:9]: `afor`?

File: README.md [hunk 8]
```diff
@@ -4,65 +4,40 @@
...
 
-> Built at @WalmartLabs powering the <http://www.walmart.com> eCommerce site, Electrode is a platform for building small to large scale universal ReactJS/NodeJS web applications with a standardized structure, best practices, and modern technologies baked in. Electrode focuses on performance, component reusability, and simple deployment to multiple cloud providers—so you can focus on what makes your app unique.
+We maintain additional code to support internal systems.
```
Comment (jchip) [inline README.md:11]: For Walmart applications, we maintain additional code to support proprietary internal systems.  If you are a Walmart developer working on internal applications, please check internal docs to get started.

**Pull request merged**